app/src/main/java/umc/cozymate/ui/my_page/UpdateMyInfoFragment.kt

```
@@ -80,13 +86,36 @@ class UpdateMyInfoFragment : Fragment() {
     }
 
+    // "yyyy-mm-dd"형식을 "yyyy년 mm월 dd일"로 바꾸는 함수
+    fun formatDate(inputDate: String): String {
+        val inputFormat = SimpleDateFormat("yyyy-MM-dd", Locale.getDefault())
+        val outputFormat = SimpleDateFormat("yyyy년 MM월 dd일", Locale.getDefault())
+        try{
+            val date = inputFormat.parse(inputDate)
+            if (date != null) {
+                return outputFormat.format(date).toString()
+            }
+        } catch (e: Exception) {
```

detekt (1.23.7)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)